xml/http_servers.xml

```diff
@@ -2542,12 +2542,15 @@
        variety of products including printers, PDUs, etc.
   -->
 
-  <fingerprint pattern="^\$ProjectRevision: 4.0.2.38 \$$">
-    <description>This banner is seen on some HP LaserJet printers.</description>
+  <fingerprint pattern="^\$ProjectRevision:[\s\w:]* ([\d\.]+) \$$">
+    <description>This banner is used to see if devices have Treck TCP/IP</description>
     <example>$ProjectRevision: 4.0.2.38 $</example>
-    <param pos="0" name="os.vendor" value="HP"/>
-    <param pos="0" name="os.device" value="Printer"/>
-    <param pos="0" name="os.family" value="LaserJet"/>
+    <example>$ProjectRevision: 4.2 $</example>
+    <example>$ProjectRevision: 6.0.1.5 $</example>
+    <example>$ProjectRevision: Last Checkpoint: 4.2.2.13 $</example>
+    <param pos="0" name="service.vendor" value="Treck"/>
```

Comment:
@tsellers-r7 what is your view on this fingerprint, is service appropriate or would service.component make more sense? Or maybe software?

Reply:
Based on a quick refresher I'd say that while this looks odd, it appears to be the `httpd` bundled with the Treck TCP/IP stack. It also aligns with how NIST has assigned the CPE for Treck so a CPE can be assigned to this fingerprint by our automation.

https://nvd.nist.gov/products/cpe/search/results?keyword=Treck&status=FINAL&orderBy=CPEURI&namingFormat=2.3